2024-04-30 08:24:33 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:33 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:40 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:40 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:43 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:43 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:46 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:46 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:50 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:50 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:53 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:53 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>

2024-04-30 08:24:57 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/mirovyye-fondovyye-indeksy?t=risers
2024-04-30 08:24:57 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555984e33cb3 <unknown>
#1 0x555984b224a7 <unknown>
#2 0x555984b6d31d <unknown>
#3 0x555984b6d3d1 <unknown>
#4 0x555984bb0734 <unknown>
#5 0x555984b8f43d <unknown>
#6 0x555984badbe9 <unknown>
#7 0x555984b8f1b3 <unknown>
#8 0x555984b6024a <unknown>
#9 0x555984b60c1e <unknown>
#10 0x555984df7fdb <unknown>
#11 0x555984dfbf2b <unknown>
#12 0x555984de40f1 <unknown>
#13 0x555984dfca92 <unknown>
#14 0x555984dc8faf <unknown>
#15 0x555984e22e08 <unknown>
#16 0x555984e22fe0 <unknown>
#17 0x555984e32e04 <unknown>
#18 0x7fbf24094ac3 <unknown>